libafl/Cargo.toml

```diff
@@ -157,6 +159,7 @@ uuid = { version = "1.4", optional = true, features = ["serde", "v4"] }
 libm = "0.2.2"
 ratatui = { version = "0.23.0", default-features = false, features = ['crossterm'], optional = true } # Commandline rendering, for TUI Monitor
 crossterm = { version = "0.27.0", optional = true }
+rand = { version = "0.8", optional = true }
```

Comment:
Why add the rand crate? Try to use our rand instead? Else, add a comment why

Reply:
?

Reply:
Managed to drop this, thanks!